test/README.md

```diff
@@ -1,4 +1,49 @@
 ### How to execute the e2e tests locally
 
-1. Start the vcluster via `devspace run dev` and running `go run -mod vendor cmd/vcluster/main.go start --sync 'networkpolicies'` in the terminal
-2. Then start the e2e tests via `VCLUSTER_SUFFIX=vcluster go test -v -ginkgo.v -ginkgo.skip='.*NetworkPolicy.*'`
+1. Start test environment:
+    - Start the vcluster in test mode `devspace run test <distro> <valuesFilePath from the testsuite>`
+    
+      - To run the test from the a general test suite i.e from `./test/e2e`.
+      
+          `devspace --namespace vcluster run test k3s --debug`
+      
+      - To run tests from a specific test suite, you'll need to specify the values file path from that test suite.
+      
+          `devspace --namespace vcluster run test k3s --var VALUES_FILE=./test/e2e_node/values.yaml --debug`
+    
+    - Then run following command in the terminal to start vcluster syncer.
+        - To run default test suite start syncer with following command
+        ```
+        go run -mod vendor cmd/vcluster/main.go start --sync 'networkpolicies' --name=vcluster --service-account=vc-workload-vcluster --kube-config-context-name=my-vcluster --leader-elect=false --sync=nodes --sync=-ingressclasses --node-selector=kubernetes.io/hostname=kind-control-plane '--map-host-service=test/test=default/test' '--map-virtual-service=test/test=test'
+        ```
+
+        - To run tests from other test suites you'll need to change the flags for `go run -mod vendor cmd/vcluster/main.go start` accordingly. You can check the list of syncer flags by running `helm template vcluster ./charts/k3s/ -f ./test/commonValues.yaml -f ./test/<test_suite>/values.yaml=`
```

Comment:
Should that `=` character be there at the end?

Reply:
Not, it's mistake.